packages/@aws-cdk-testing/cli-integ/package.json

```diff
@@ -39,10 +39,22 @@
   },
   "dependencies": {
     "@octokit/rest": "^18.12.0",
-    "aws-sdk": "^2.1610.0",
+    "@aws-sdk/client-codeartifact": "3.421.0",
+    "@aws-sdk/client-cloudformation": "3.421.0",
+    "@aws-sdk/client-s3": "3.421.0",
+    "@aws-sdk/client-ecr": "3.421.0",
+    "@aws-sdk/client-ecs": "3.451.0",
```

Comment:
This is really weird. If we mention `3.421.0` it leads to test errors in `packages/@aws-cdk/custom-resource-handlers/test/aws-events-targets/aws-api-handler/aws-api-handler.test.ts`. Unfortunately, we had set the client version to `3.451.0` in there. 

I see there is a lot of mismatch between sdk versions in different `package.json`s and we need to align them to same version. We cannot lower them to a previous version, so probably will end up aligning to the version that is the highest in our package.